services/web/server/src/simcore_service_webserver/users/_handlers.py

```diff
@@ -202,3 +205,16 @@ async def mark_notification_as_read(request: web.Request) -> web.Response:
             raise web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
 
     raise web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
+
+
+@routes.get(f"/{API_VTAG}/me/permissions", name="list_user_permissions")
+@login_required
+@permission_required("user.permissions.read")
+async def list_user_permissions(request: web.Request) -> web.Response:
+    req_ctx = _RequestContext.parse_obj(request)
+    list_permissions: list[Permission] = await api_list_user_permissions(
+        request.app, req_ctx.user_id, req_ctx.product_name
+    )
+    return envelope_json_response(
+        [PermissionGet.construct(**p.dict()) for p in list_permissions]
```

Comment:
NOTE: when using `construct` it is important to pass the argument `_fields_set` since `envelope_json_response` configuration might need that information to drop or not certain fields when exporting. If `list_permissions` is a list of the original models, the idea is to "transfer" not only the data but also the "metadata" between models. 

SEE an example in https://docs.pydantic.dev/latest/usage/models/#creating-models-without-validation

Reply:
ok